src/common/integrations/GoogleDrive.js

```diff
@@ -86,6 +86,11 @@ class GoogleDriveUtil {
 
     login = () => {
         gapi.client.setToken('');
+        (this.client.callback = response => {
+            this.accessToken = response.access_token;
+            localStorage.setItem('accessToken', this.accessToken);
+            this.updateLoginStatus(true);
+        }),
```

Comment:
@vinu-deriv why should we use here ( funcName => {...} )`,` instead of `; `? This function is followed by `this.client.requestAccessToken({ prompt: '' });`...
Maybe i something didn't get, could you please explain?
Maybe here should be:
```
login = () => {
        gapi.client.setToken('');
        (this.client.callback = response => {
            this.accessToken = response.access_token;
            localStorage.setItem('accessToken', this.accessToken);
            this.updateLoginStatus(true);
        });
        this.client.requestAccessToken({ prompt: '' });
};
```

no?